pyfar/classes/filter.py

```diff
@@ -383,6 +418,35 @@ def init_state(self, cshape, state='zeros'):
                 new_state[idx, ...] = spsignal.lfilter_zi(coeff[0], coeff[1])
         super().init_state(state=new_state)
 
+    def impulse_response(self, n_samples=None):
+        """
+        Compute the finite impulse response of the filter.
+
+        Parameters
+        ----------
+        n_samples : int, str, optional
```

Comment:
```suggestion
        n_samples : int, optional
```
I don't think we can pass strings here, can we? If we do, mention which strings are possible in the explanation.